packages/api-gateway/src/drivers/http/adapters/users/updateUser.js

```diff
@@ -0,0 +1,92 @@
+const errorHandler = require('./errorHandler');
+
+async function updateUser(req, reply) {
+  try {
+    const {
+      email,
+      avatar,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      nationality,
+      dniId,
+      dniFrontImg,
+      dniBackImg,
+      gender,
+      phoneNumber,
+      emergencyNumber,
+      passport,
+      address: {
+        country, city, state, address, zip,
+      },
+    } = req.body;
+    const { userId } = req.params;
+
+    req.log.info('[http-server]: Creating user with: ', {
+      userId,
+      email,
+      avatar,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      nationality,
+      dniId,
+      dniFrontImg,
+      dniBackImg,
+      gender,
+      phoneNumber,
+      emergencyNumber,
+      passport,
+      address: {
+        country,
+        city,
+        state,
+        address,
+        zip,
+      },
+    });
+
+    const result = await this.userServices.updateUser({
```

Comment:
I think we could pass the user info object directly and left the service decide how it will use the info